packages/alert/src/js/alerts.js

```diff
@@ -46,22 +46,22 @@ class Alerts {
       if (!alert.showOnPages.length) {
         return true;
       }
+      const href = window.location.href.replace(/\/+$/, '');
+      const hrefMatches = (page) => {
+        const expression = `${page}`
+          .replace(/\/+$/, '')
```

Comment:
Ah, so the JS version of preq_quote was also quoting the / ?

Reply:
@larowlan no I'm stripping the trailing slash from both the href and the user supplied path because content authors are inconsistent as to whether they include it or not.

Reply:
Gotcha